README.md

```diff
@@ -1,87 +1,237 @@
-let g:LanguageClient_serverCommands = {
-\ 'text': ['unified-language-server', '--parser=retext-english', '--stdio'],
-\ 'markdown': ['unified-language-server', '--parser=remark-parse', '--stdio'],
-\ }
+createUnifiedLanguageServer({
+  ignoreName: '.remarkignore',
+  packageField: 'remarkConfig',
+  pluginPrefix: 'remark',
+  plugins: ['remark-parse', 'remark-stringify'],
+  rcName: '.remarkrc'
+})
 ```
 
-And you’re ready to go!
+That’s all there is to it.
+You have just created a language server for remark.
 
-## Configuration
+## API
 
-The server has default configurations for `remark-parse`(for  markdown) and
-`remark-english` (for text):
+### `createUnifiedLanguageServer(options)`
 
-```json
-{
-  "retext-english": {
-    "plugins": [
-      ["#retext-profanities"],
-      ["#retext-spell", "#dictionary-en-gb"]
-    ],
-  },
-  "remark-parse": {
-    "plugins": [
-      ["#remark-preset-lint-markdown-style-guide"]
-      ["#remark-retext", "#parse-latin"],
-      ["#retext-profanities"],
-      ["#retext-spell", "#dictionary-en-gb"]
-    ]
-  }
-}
-```
+Create a language server for a unified ecosystem.
+
+##### `options`
+
+Configuration for `unified-engine` and the language server.
+
+###### `options.defaultSource`
+
+Default source used for diagnostics (`string`, optional)
+
+###### `options.ignoreName`
+
+Name of ignore files to load (`string`, optional)
+
+###### `options.packageField`
+
+Property at which configuration can be found in package.json files (`string`,
+optional)
+
+###### `options.pluginPrefix`
+
+Optional prefix to use when searching for plugins (`string`, optional)
+
+###### `options.plugins`
+
+Plugins to use by default (`Array|Object`, optional)
+
+Typically this contains 2 plugins named `*-parse` and `*-stringify`.
+
+###### `options.rcName`
+
+Name of configuration files to load (`string`, optional)
+
+## Examples
+
+For examples, see the following projects:
+
+*   [redot-language-server][]
+*   [rehype-language-server][]
+*   [remark-language-server][]
+
+## Types
+
+This package is fully typed with [TypeScript][].
+It exports an `Options` type, which specifies the interface of the accepted
+options.
+
+## Compatibility
+
+Projects maintained by the unified collective are compatible with all maintained
+versions of Node.js.
+As of now, that is Node.js 12.20+, 14.14+, and 16.0+.
+Our projects sometimes work with older versions, but this is not guaranteed.
+
+This plugin works with `unified` version 10+.
+
+## Related
+
+*   [unified][]
+    — create pipeline for working with syntax trees
+*   [unified-args][]
+    — create a CLI for a unified pipeline
+
+## Contribute
+
+See [`contributing.md`][contributing] in [`unified/.github`][health] for ways
+to get started.
+See [`support.md`][support] for ways to get help.
+
+This project has a [code of conduct][coc].
+By interacting with this repository, organization, or community you agree to
+abide by its terms.
+
+## License
+
+[MIT][license] © [Titus Wormer][author]
```

Comment:
Should I just use my own name here? Or @aecepoglu?

Reply:
AFAIK we keep the original author. For instance, [`react-markdown`](https://github.com/remarkjs/react-markdown#license) has basically been redone from the ground up but the original author is still in the license. I don't have a strong opinion on this but that's how it seems to be done most of the time.